utils/streams/src/main/java/io/apicurio/registry/utils/streams/diservice/AsyncBiFunctionServiceGrpcClient.java

```diff
@@ -51,6 +55,8 @@ public void close() {
         ByteString keyByteStr = keyBytes == null ? ByteString.EMPTY : ByteString.copyFrom(keyBytes);
         ByteString reqByteStr = reqBytes == null ? ByteString.EMPTY : ByteString.copyFrom(reqBytes);
 
+        logger.info("Performing call through GRPC for key: {}", key);
```

Comment:
Probably need to change this to debug or trace level if it's going to happen for every GRPC call.